### ISSUE TITLE
cloneElement audit

### ISSUE THREAD

**Issue by qaisjp** [multitheftauto/mtasa-blue#404]
**Is your feature request related to a problem? Please describe.**
cloneElement does not migrate many properties. This needs to be fixed.

**Describe the solution you'd like**

A run through of every cloneable entity, making sure that every property can be cloned.

**Stretch goals**

We should try and figure out a nice way to track these properties that are cloneable. We're most likely going to have to resort to preprocessor directives but it would be cool if we could somehow tag member fields.

Check out https://www.codesynthesis.com/~boris/blog/2012/04/18/cxx11-generalized-attributes/?

This may be out of the scope of this issue but it is definitely a "nice to have". Let's also use this issue to discuss the stretch goal.

**Additional context**
Blocked by #182.
Related to #403.
An expansion of https://bugs.mtasa.com/view.php?id=8465. Please read this issue for details relating to **vehicle handling**
